apps/aecore/src/aec_db_gc.erl

```diff
@@ -0,0 +1,206 @@
+-module(aec_db_gc).
```

Comment:
Please add portions of the report as documentation here, since this is also where it will be read.

Reply:
Added small section linking to the paper if anybody wants to read it.